docs/pages/headeraudit.md

```diff
@@ -0,0 +1,24 @@
+# Cybersecurity Task 1 - HTTP Header Audit
+
+There are many free tools available that can help us audit the security standards of a given site. 
+A very common, yet overlooked security standard is the HTTP Headers that a site transfers when a user accesses their site.
+[SecurityHeaders.com](https://securityheaders.com/) is one great free tool that we can use. 
+It gives a letter grade for a URL, as well as gives details of what can be fixed and what can be improved.
+
+## Part 1 - Report
+Analyze the security standards implemented on our [project website](https://scarletstudio.github.io/transithealth/) using the free header checking tool [SecurityHeaders.com](https://securityheaders.com).  
+
+- Describe each header and how it applies to the usage of our site.  It’s possible some headers are not relevant to our site.  Ask if you are not sure.
+- Determine which headers are missing
+- Determine which headers are present, but could be improved (if any)
+- Determine which headers are properly implemented.
+- Assess & describe your findings in a report that is written in a jupyter notebook.
```

Comment:
Do you think the report should be in Jupyter or in a Google Doc? I think a doc might be easier to leave feedback on, and we don't necessarily need to run code in a notebook as part of the report, perhaps separately.